test/test.js

```diff
@@ -0,0 +1,50 @@
+const assert = require('assert');
```

Comment:
Perhaps this should be written in Ruby rather than take a dependency on another language/ecosystem?

(Happy to accept this in its current form and then I can port it.)

Reply:
Didn't even cross my mind as I have not written a lick of Ruby.

If you port it, I will have enough context to learn what I need to quickly and will keep it to Ruby in the future. Good call.